Makefile

```diff
@@ -58,7 +58,7 @@ build-slim:
 test: run-unit-tests
 
 .PHONY: lint
-lint:
+lint: build-slim
```

Comment:
you should also compile all tests before linting. This can be done by adding `build-test` target.